### ISSUE TITLE
Evil-Droid automatically exits

### ISSUE THREAD

**Issue by insidevoid** [M4sc3r4n0/Evil-Droid#28]
after launching evil-droid
it runs until verifying the dependencies, then it stops the apache2 server and postgresql server and exits.

**Comment by Arkamis** [comment]
Yeah, i have the same issue, it have to be something with this line:
`start=$(zenity --question --title="☢ Evil-Droid Framework ☢" --text "Execute framework and Services?" --width 270 2> /dev/null)
  if [ "$?" -eq "0" ]; then
    apache_svc_start
    postgresql_start
`

**Comment by xfactor67** [comment]
Same problem bro how to fix it please tell me

**Comment by Arkamis** [comment]
D

> Same problem bro how to fix it please tell me

Hi bro, It have been a while since I worked with evil-driod, but  with this issue you can just change the file in that specific line to always return true. I think that's how i fixed it,
